### PR TITLE
091024-1610

### PR DESCRIPTION
**Pull request by ianpark318**

### CODE REVIEW

File: assets/img/team/heeju_han.jpg
Comment (zzangjinsun) [inline assets/img/team/heeju_han.jpg:1]: Please resize profile images to 150 x 200

File: team.md [hunk 1]
```diff
@@ -65,7 +40,7 @@ Prof. Jinsun Park is an assistant professor at School of Computer Science and En
         <td width="38%" style="border-left: none; border-right: none; border-top: none; border-bottom: none;">
             <span style="font-size: bigger; font-weight: bold">Janghyun Kim (김장현) 👑</span> <br>
             <span style="font-size: smaller;">📫 jangjoa4497_at_gmail.com</span> <br>
-            <span style="font-size: smaller;">🔎 Depth Estimation / Completion</span>
+            <span style="font-size: smaller;">🔎 Depth Perception & Multi-sensor Fusion</span>
```
Comment (zzangjinsun) [inline team.md]: Multi-Sensor

File: team.md [hunk 2]
```diff
@@ -94,7 +76,7 @@ Prof. Jinsun Park is an assistant professor at School of Computer Science and En
         <td width="38%" style="border-left: none; border-right: none; border-top: none; border-bottom: none;">
             <span style="font-size: bigger; font-weight: bold">Seong Heon Ha (하성헌)</span> <br>
             <span style="font-size: smaller;">📫 123skfro_at_pusan.ac.kr</span> <br>
-            <span style="font-size: smaller;">🔎 3D Semantic Segmentation</span>
+            <span style="font-size: smaller;">🔎 Sensor Fucsion & Sign Language Recognition</span>
```
Comment (zzangjinsun) [inline team.md]: Fucsion -> Fusion

**Review by zzangjinsun — CHANGES_REQUESTED** [review]
Please refer to comments to revise this PR

File: assets/img/team/heeju_han.jpg
Comment (ianpark318) [reply assets/img/team/heeju_han.jpg:1]: Done

**Pull request merged**